README.md

```diff
@@ -23,19 +23,32 @@ yarn add @BolajiOlajide/now-playing
 import { NowPlaying, Providers } from '@BolajiOlajide/now-playing'
 
 const np = new NowPlaying(Providers.SPOTIFY, {
-  useCache: false, // default is true
-  cacheDuration: 30000, // in milliseconds
+  useCache: false, // (optional) default is true
+  cacheDuration: 30000, // (optional) in milliseconds
   streamerArgs: {
     clientId: 'foo',
     clientSecret: 'bar',
     refreshToken: 'baz',
   },
+  storer: // (optional) custom
```

Comment:
Also what do you think about exposing the `IStorer` interface? If we do that, we can update the example to something like this.

```ts
import { NowPlaying, Providers, IStorer } from '@BolajiOlajide/now-playing'

let myCustomStorer: IStorer

const np = new NowPlaying(Providers.SPOTIFY, {
  useCache: false, // (optional) default is true
  cacheDuration: 30000, // (optional) in milliseconds
  streamerArgs: {
    clientId: 'foo',
    clientSecret: 'bar',
    refreshToken: 'baz',
  },
  storer: myCustomStorer // (optional) custom storage implementation
})
```

Reply:
Gotcha! THis is clearer I guess.

Reply:
Done.